packages/client/src/app/shared/global-constants.ts

```diff
@@ -1,3 +1,26 @@
 export class GlobalConstants {
```

Comment:
I think you can get away with just calling this `Constants`; I'd say the "Global" is implied.